projects/demo/src/modules/components/select/index.html

```diff
@@ -0,0 +1,298 @@
+
+                    <div>
+                        For the most cases
+                        <code>&lt;tui-data-list-wrapper [itemContent]="..." /&gt;</code>
+                        can be enough for your task.
+                    </div>
+
+                    <div>
+                        Explore
+                        <a
+                            fragment="content-customization"
+                            routerLink="."
+                            tuiLink
+                        >
+                            this example
+                        </a>
+                        for more details.
+                    </div>
+                </tui-notification>
+            </ng-template>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo [control]="control">
+            <ng-template>
+                <tui-textfield
+                    tuiChevron
+                    [content]="control.value && textfieldContent"
+                    [tuiDropdownAlign]="dropdown.align"
+                    [tuiDropdownAppearance]="dropdown.appearance"
+                    [tuiDropdownDirection]="dropdown.direction"
+                    [tuiDropdownLimitWidth]="dropdown.limitWidth"
+                    [tuiDropdownMaxHeight]="dropdown.maxHeight"
+                    [tuiDropdownMinHeight]="dropdown.minHeight"
+                    [tuiDropdownOffset]="dropdown.offset"
+                    [tuiTextfieldCleaner]="textfieldDoc.cleaner"
+                    [tuiTextfieldSize]="textfieldDoc.size"
+                >
+                    <input
+                        placeholder="Select something"
+                        tuiSelect
+                        [formControl]="control"
+                        [invalid]="controlDoc.invalid"
+                        [readOnly]="controlDoc.readonly"
+                        [tuiDisabled]="controlDoc.disabled"
+                    />
+
+                    <tui-data-list-wrapper
+                        *tuiTextfieldDropdown
+                        new
+                        [items]="items"
+                    />
+                </tui-textfield>
+            </ng-template>
+        </tui-doc-demo>
+        <table tuiDocAPI>
+            <tbody
+                #textfieldDoc
+                tuiDocTextfield
+            >
+                <!--TODO: create the inherited documentation for Textfield [content] prop)-->
+                <tr
+                    name="[content]"
+                    tuiDocAPIItem
+                    type="PolymorpheusContent"
+                    [items]="textfieldContentVariants()"
+                    [(value)]="textfieldContent"
+                >
+                    A template for custom view of textfield value.
+                </tr>
+            </tbody>
+
+            <tbody
+                #dropdown
+                tuiDocDropdown
+                [hiddenOptions]="['open', 'tuiDropdownEnabled']"
```

Comment:
Why hiding `open`? People should know how to control dropdown for selects manually

Reply:
https://github.com/taiga-family/taiga-ui/pull/10642/commits/f83d1f83aa343cdf442eebaf72b4d169d1fdc43a